### PR TITLE
Sro webui

### PR DESCRIPTION
**Pull request by sro-csis**
changelog:

* colors were changed to make it more colorblind friendly

* 'ok' jobs are hidden by default (the entire list can be expanded by clicking on the header)

### CODE REVIEW

**Comment by sro-csis** [comment]
@car-csis can you take a look again?

File: src/proccer/static/proccer.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        $(role).find('.toggle-text').text('[show all ' + n_jobs +' jobs]');
+      } else {
+        // if there are not hidden jobs
+        $(role).find('.toggle-text').text('[hide]');
+      }
+    }
+
+    // Hides all jobs with status 'ok' and updates the header text
+    var hide_ok = function(role) {
+      $(role).find('.role-details:not(:has(.job:not(.ok)))').hide();
```
Comment (mlh-csis) [inline src/proccer/static/proccer.js]: No expert here, but is the double negation here a result of css-selector shortcomings or could it be expressed more simply? Seems rather convoluted..

**Comment by ski-csis** [comment]
Can you use non-minified jQuery?

File: src/proccer/templates/job.html [hunk 2]
```diff
@@ -8,7 +8,9 @@
 </head>
 <body class='job-section'>
   <div class='box'>
-    <h1 class='${job.state}'>Job: ${job}</h1>
+    <div class='header ${job.state}'>
+    <h1>${job}</h1>
```
Comment (ski-csis) [inline src/proccer/templates/job.html]: Indentation.

File: src/proccer/static/style.css [hunk 3]
```diff
@@ -45,26 +45,41 @@ td.number {
...
-    border-radius: 10px 10px 0 0;
+    padding-right: 1em;
+    border-radius: 10px 10px 10px 10px;
+}
+.box .header h1 {
+    display: inline-block;
+    margin: .3em;
+}
+.box .header:hover {
+    cursor:pointer;
```
Comment (ski-csis) [inline src/proccer/static/style.css]: Space after colon.

File: src/proccer/static/proccer.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
+(function() {
+  $(document).ready(function() {
+
+    // Updates the text on the roll header.
+    // If there are hidden jobs -> '[show all X jobs]'
+    // otherwise -> '[hide]'
+    var update_text = function(role) {
+      if ($(role).find('.job:not(:visible)').length > 0) {
```
Comment (ski-csis) [inline src/proccer/static/proccer.js]: Indentation in JS is four spaces.

**Comment by sro-csis** [comment]
@ski-csis can you take another look, Sune?
I'll squash after a + 1.

**Comment by ski-csis** [comment]
:+1: Supergreen!

**Pull request merged**